SEED-platform/seed
# 4635 geojson endpoint

#### Any background context you want to provide?

#### What's this PR do?
Adds a public geojosn endpoint using existing functionality from the inventory export endpoint.
Adds an org setting for public_geojson_enabled
Moves some tax_lot_property functions to a reusable util.

#### How should this be manually tested?
With properties that have geo data like lat/long,
1. Test with endpoints disabled. Go to org settings and unselect either "enable public endpoints" or "enable public geojson". Go to /api/v3/public/organizations/{org_id}/cycles/{cycle_id}/geo.json. The endpoints should return an error message

2. Test with endpoints enabled. Go to /api/v3/public/organizations/{org_id}/cycles/{cycle_id}/geo.json. Copy the json response and paste into [geojson.io](geojson.io). Map should show your seed properties


#### What are the relevant tickets?
#4635

#### Screenshots (if appropriate)
![Screenshot 2024-05-03 at 11 22 19 AM](https://github.com/SEED-platform/seed/assets/58446472/10a8d0ab-d650-43d6-b4bb-f747d58781b1)
![Screenshot 2024-05-03 at 11 18 56 AM](https://github.com/SEED-platform/seed/assets/58446472/c8821a52-75e9-474d-a3fa-65f176b38216)
